CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [5.4.0] - 2023-06-13
 
+### Added
+
+- Loggign the command invkoing the run-script
+
```

Comment:
Nice thought, though we don't typically update the changelog until we make a release. The `5.4.0` release has already gone out the door :).

So please remove this change and I'll be sure to mention it the next time we release.